### PR TITLE
After sending donation 2 toast displays

### PR DESCRIPTION
**Pull request by johnpaul-sun**
## Issue Link
- [Backlog link](https://framgiaph.backlog.com/view/SPH_SOLIDITY-76)

## Target Page | Component | Function
- http://localhost:3000/campaigns/

## Pre-condition
-

## Steps | Commands
- git pull
- git checkout bug/sol-76

## Expected Output
- It should show a single toast message for a successful donation

## Definition of Done
- It should show a single toast message for a successful donation

## Test View Points
- [ ] Add here

## Notes
N/A

## Screenshots | Recordings
image

### General
- [ ] Does the code meet the requirements of the project?
- [ ] Is the code readable and maintainable?
- [ ] Are there any unnecessary or redundant lines of code?
- [ ] Are there any long methods or functions that should be refactored?
- [ ] Are there any performance issues?

### Functionality
- [ ] Are there any logical errors?
- [ ] Are edge cases handled properly?
- [ ] Are there any potential security vulnerabilities?

### Code Style
- [ ] Does the code adhere to the project's coding standards?
- [ ] Are variable and function names descriptive and meaningful?
- [ ] Is there proper use of comments and documentation?
- [ ] Is the code properly formatted? Are indentation, spacing, and formatting consistent?

### Security
- [ ] Is the website secure from common attacks?
- [ ] Are inputs properly validated and sanitized?
- [ ] Are sensitive data and credentials properly protected?

### Testing
- [ ] Are there sufficient test cases?
- [ ] Do the test cases cover all functionality?
- [ ] Are the test cases properly written and documented?

### Additional Comments
- [ ] Are there any other issues or concerns with the code?
- [ ] Are there any suggestions for improvement?
- [ ] Are there any positive aspects of the code that should be highlighted?

### CODE REVIEW

File: fe/src/plugins/ethers.ts [hunk 1]
```diff
@@ -54,7 +55,13 @@ export default defineNuxtPlugin(async () => {
 
   smartContract?.on("DonationSent", (sender) => {
     if (sender.toLowerCase() === ethereum.selectedAddress.toLowerCase()) {
-      toast.success(`Fund successfully send!`);
+      if (!isListenerExecuted) {
```
Comment (chevyB) [inline fe/src/plugins/ethers.ts]: To assignee and code reviewer : Pls check another way to avoid duplicate toast if possible

**Review by kenorenzsun — DISMISSED** [review]

**Review by chevyB — DISMISSED** [review]

**Review by arden-dave-sun — APPROVED** [review]

**Comment by ipilikent** [comment]
## Approved PR
-  should show a single toast message for a successful donation

## Proof
<details>

Sol- 28 Bugfix Toast

</details>

-----
- Total Test time:  15 mins.
- Total Comments: 0
- Total Iterations: 0

**Comment by ipilikent** [comment]
## Approved PR
-  A single toast message for a successful donation is displayed

## Proof
<details>

Sol- 28 Bugfix Toast

</details>

-----
- Total Test time:  0.15 mins.
- Total Comments: 0
- Total Iterations: 0

**Review by ipilikent — APPROVED** [review]

**Review by wylenjoan — APPROVED** [review]
Approved :white_check_mark:
Review time: 15 mins
Comment: 0
Iteration: 0
Working:
<details>
<summary> Only one toast displays after sending donation </summary>

image

</details>

**Pull request merged**